libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/route/ReplayRouteMapper.kt

```diff
@@ -28,19 +26,19 @@ class ReplayRouteMapper @JvmOverloads constructor(
     var options: ReplayRouteOptions = ReplayRouteOptions.Builder().build()
 ) {
 
-    private val tag = Tag("MbxReplayRouteMapper")
     private val replayRouteDriver = ReplayRouteDriver()
-    private var logger: Logger? = null
 
     /**
      * @see ReplayRouteMapper
      */
+    @Deprecated(
+        message = "Setting a Logger has no effect, logging is handled internally.",
+        replaceWith = ReplaceWith("ReplayRouteMapper()", "ReplayRouteMapper(options)")
```

Comment:
the second param is `imports`, right? It should be already imported in place where it was previous used
```suggestion
        replaceWith = ReplaceWith("ReplayRouteMapper()")
```

Reply:
I added the second replace with because the `constructor` is overloaded but both options are valid to replace the one with the `@Deprecated` `Logger`.

Reply:
I see. It's not working: second and every next param is about `imports`, see signature 
<img width="749" alt="image" src="https://user-images.githubusercontent.com/8738058/156392478-a0db7948-e5a5-40c1-9a23-ae2e17078387.png">

Reply:
BTW What's `imports`? 🤔